src/js/components/Markdown/Markdown.js

```diff
@@ -15,36 +15,44 @@ import { TableHeader } from '../TableHeader';
 import { TableRow } from '../TableRow';
 import { MarkdownPropTypes } from './propTypes';
 
-const GrommetMarkdown = ({ components, options, theme, ...rest }) => {
-  const heading = [1, 2, 3, 4].reduce((obj, level) => {
-    const result = { ...obj };
-    result[`h${level}`] = {
-      component: Heading,
-      props: { level },
-    };
-    return result;
-  }, {});
+const GrommetMarkdown = forwardRef(
+  ({ components, options, theme, ...rest }, ref) => {
+    const heading = [1, 2, 3, 4].reduce((obj, level) => {
+      const result = { ...obj };
+      result[`h${level}`] = {
+        component: Heading,
+        props: { level },
+      };
+      return result;
+    }, {});
 
-  const overrides = deepMerge(
-    {
-      a: { component: Anchor },
-      img: { component: Image },
-      p: { component: Paragraph },
-      table: { component: Table },
-      td: { component: TableCell, props: { plain: true } },
-      tbody: { component: TableBody },
-      tfoot: { component: TableFooter },
-      th: { component: TableCell },
-      thead: { component: TableHeader },
-      tr: { component: TableRow },
-    },
-    heading,
-    components,
-    options && options.overrides,
-  );
+    const overrides = deepMerge(
+      {
+        a: { component: Anchor },
+        img: { component: Image },
+        p: { component: Paragraph },
+        table: { component: Table },
+        td: { component: TableCell, props: { plain: true } },
+        tbody: { component: TableBody },
+        tfoot: { component: TableFooter },
+        th: { component: TableCell },
+        thead: { component: TableHeader },
+        tr: { component: TableRow },
+      },
+      heading,
+      components,
+      options && options.overrides,
+    );
 
-  return <Markdown options={{ ...options, overrides }} {...rest} />;
-};
+    // We have to wrap in a div because markdown-to-jsx is a
+    // functional component that doesn't handle refs.
+    return (
+      <div ref={ref}>
+        <Markdown options={{ ...options, overrides }} {...rest} />
```

Comment:
I tried setting options.wrapper= props => <div ref={ref} {...props} />. It definitely keeps the one <div> wrapper. I didn't verify the ref worked as expected.

It's hard to tell from the docs if that would potentially get used somewhere other than the toplevel.